code/traffic-register/traffic-register.css

```diff
@@ -454,25 +454,52 @@ a.ang-link:link {text-decoration: none;}
-#kn-scene_588 #view_1476 div.kn-detail:hover { opacity: 0.9; cursor: pointer; border-color: #dd4d4d; border-width: 2px; min-width: 190px !important; max-width: 27% !important; }
-#kn-scene_588 #view_1476 .kn-detail-label { background-color: transparent; min-width: 0% !important; }
+#kn-scene_588 #view_1476 div.kn-detail:hover { opacity: 0.9; cursor: pointer; border-color: #dd4d4d; border-width: 2px; }
+#kn-scene_588 #view_1476 .kn-detail-label { background-color: transparent; min-width: 0% !important; } */
 
-/*Reg Doc Details Header navigating from Regulations Search*/
-#kn-scene_773 #view_1592 div.kn-detail { padding: 1px 2px; border-style: solid; border-width: 1px; border-color: #dd4d4d; border-radius: 8px; 
+/*Reg Doc Details Header navigating from Document Search - Historical SharePoint PDF*/
+/* #kn-scene_588 #view_1706 div.kn-detail { padding: 1px 2px; border-style: solid; border-width: 1px; border-color: #163f6e; border-radius: 8px; 
   box-shadow: 0px 0px 0px 0px gray; font-size: 1.25em; background-color: #ebebeb; color: #163f6e; text-align: left; min-width: 190px !important; max-width: 27% !important; }
-#kn-scene_773 #view_1592 div.kn-detail:hover { opacity: 0.9; cursor: pointer; border-color: #dd4d4d; border-width: 2px; min-width: 190px !important; max-width: 27% !important; }
-#kn-scene_773 #view_1592 .kn-detail-label { background-color: transparent; min-width: 0% !important; }
+#kn-scene_588 #view_1706 div.kn-detail:hover { opacity: 0.9; cursor: pointer; border-color: #163f6e; border-width: 2px; }
+#kn-scene_588 #view_1706 .kn-detail-label { background-color: transparent; min-width: 0% !important; } */
+
+/*Reg Doc Details Header navigating from Regulations Search - Generated Power BI PDF*/
+/* #kn-scene_773 #view_1592 div.kn-detail { padding: 1px 2px; border-style: solid; border-width: 1px; border-color: #dd4d4d; border-radius: 8px; 
+  box-shadow: 0px 0px 0px 0px gray; font-size: 1.25em; background-color: #ebebeb; color: #163f6e; text-align: left; min-width: 190px !important; max-width: 27% !important; }
+#kn-scene_773 #view_1592 div.kn-detail:hover { opacity: 0.9; cursor: pointer; border-color: #dd4d4d; border-width: 2px; }
+#kn-scene_773 #view_1592 .kn-detail-label { background-color: transparent; min-width: 0% !important; } */
+
+/*Reg Doc Details Header navigating from Regulations Search - Historical SharePoint PDF*/
+/* #kn-scene_773 #view_1705 div.kn-detail { padding: 1px 2px; border-style: solid; border-width: 1px; border-color: #163f6e; border-radius: 8px; 
+  box-shadow: 0px 0px 0px 0px gray; font-size: 1.25em; background-color: #ebebeb; color: #163f6e; text-align: left; min-width: 190px !important; max-width: 27% !important; }
+#kn-scene_773 #view_1705 div.kn-detail:hover { opacity: 0.9; cursor: pointer; border-color: #163f6e; border-width: 2px; }
+#kn-scene_773 #view_1705 .kn-detail-label { background-color: transparent; min-width: 0% !important; } */
 
```

Comment:
Previous 4 code blocks above have been commented out with the 2 new code blocks under the 'Style PDF Link Fields as Buttons' section